esp32/door-sensor/src/config-example.h

```diff
@@ -0,0 +1,8 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+const char *WIFI_SSID = "your_wifi_ssid";
+const char *WIFI_PASSWORD = "your_wifi_password";
+const char *SERVER_URL = "http://your_server_ip:8080/update_door_status";

```

Comment:
Enable these lines to be able to be set via defines on the command line
```suggestion
#ifdef WIFI_SSID_DEFINE
const char *WIFI_SSID = WIFI_SSID_DEFINE;
#else
const char *WIFI_SSID = "your_wifi_ssid";
#endif
const char *WIFI_PASSWORD = "your_wifi_password";
const char *SERVER_URL = "http://your_server_ip:8080/update_door_status";
```

Reply:
https://github.com/GryphonRacingFSAE/Discord-Bot/pull/20/commits/2524b77d8156b81254e08da056a6d3c8688cdb32